packages/manager/apps/dedicated/client/app/billing/autoRenew/actions/activation/activation.module.js

```diff
@@ -1,10 +1,23 @@
+import angular from 'angular';
+import 'angular-translate';
+import ngAtInternet from '@ovh-ux/ng-at-internet';
+import '@ovh-ux/ng-translate-async-loader';
```

Comment:
Why sometimes `import foo from 'bar'` and sometimes `import 'bar'` ? (apart from when there is no choice)

Reply:
copy paste :/
i know that some older modules doesn't export moduleName but in that case we should be able to ```import foo from bar;``` i'm going to fix it